---
title             : "The Relationship between Life Phase-, Cell Type- and Tissue-Specific Transcriptome, and Psychiatric and Neurological Disorders."
shorttitle        : "Life Phase-, Cell Type- and Tissue-Specific Transcriptome & Disorders"

author: 
  - name          : "Tim Finke"
#    corresponding : yes
#    address       : "Postal address"
#    email         : " "
    institution   : "Vrije Universiteit Amsterdam"


affiliation:
  - id            : "1"
    institution   : "Vrije Universiteit Amsterdam"

note: |

  Thesis Internship I Genes in Behaviour and Health
  
  Student number: XXXXXX
  
  Supervisor and first assessor: XXXXXXX
  
  Second assessor: XXXXXXXXX
  
abstract: |
  Genetic influences on psychiatric disorders transcend diagnostic boundaries, indicating that contributing loci confer risk for multiple disorders. This study aimed to identify genetic influences on psychiatric and neurological disorders, indicating pleiotropy of contributing loci. Pleiotropy was mapped for 12 psychiatric and neurological disorders by estimating genetic correlations. Furthermore, we identified genes that are part of the brain tissue-, life phase-, and cell type-specific transcriptome of disorders. Identifying these genes was done by partitioning heritability through linkage disequilibrium score regression. Height was used as a negative control. Genetic relationships found were largely concordant with earlier research, with some new insights found due to the addition of other psychiatric and neurological disorders. A multitude of genes expressed in brain tissue-, life phase-, and cell type-specific transcriptome was found to be enriched in their effect on psychiatric and neurological disorders. Found results may hold implications for diagnostic definitions of disorders and put into question how valid these and other phenotypes are if solely based upon observational research.
  
keywords          : 
wordcount         : "X"

bibliography      : ["r-references.bib", "articles.bib"] 
nocite            : ["@1000GPC2015", "@IHM3C2010"]

floatsintext      : no
figsintext        : yes
figurelist        : no
tablelist         : no
footnotelist      : no
linenumbers       : no
mask              : no
draft             : no

always_allow_html : TRUE

documentclass     : "apa6"
classoption       : "man"
output            : papaja::apa6_pdf
---
```{r setup, include=FALSE}
options(knitr.duplicate.label = "allow")

knitr::opts_chunk$set(
  figure.width  = 10, 
  figure.height = 10, 
  figure.path   = 'images/',
  echo       = FALSE, 
  warning    = FALSE, 
  message    = FALSE,
  cache      = FALSE)
library("papaja")
library("SummarizedExperiment")
library("DESeq2")
library("tidyverse")
library("data.table")
library("kableExtra")
library("corrplot")
library("Hmisc")
library("qgraph")
library("qvalue")     
library("cowplot")
library("Hmisc")
library("igraph")
library("RColorBrewer")

source("Script Functions.R")
#source("Script Data Table1.R")
# takes waaaaay too long (time indications present in script), I copied the column for table 1 (Number of significant GWAS loci) by hand. However the code which would run this automatically is still present in the corresponding chunk for reference!
source("script results CTS_analyses.R")
#source("ldsc network plots.R")

# table genetic correlation LDSC
table_results <- read.table(file = file.path(path_data_rg_matrix, "collected_results.tab"), header = TRUE)

# Bonferroni corrected significance level for genetic correlations
sig_level_corrs <- 0.05 / (((12 * 12) / 2) - 12)
```
# Introduction

Both twin and large-scale genomic studies have established that there is an influence of genetic variation on risk for a broad range of psychiatric disorders [@Smoller2019]. Psychiatric disorders are highly polygenic, which means that contributing variants must influence disorders in combination with many other variants [@Strachan2019]. Therefore a large proportion of heritability is contributed by common variation, only marginally increasing or decreasing risk of disease. Genome-wide association studies (GWAS) of attention-deficit/hyperactivity disorder (ADHD), Alzheimer’s disease (ALZ), anorexia nervosa (AN), autism spectrum disorder (ASD), bipolar disorder (BIP), lifetime cannabis use (CAN), insomnia (INS), major depression (MD), neuroticism depressed affect (NEUdep), neuroticism worry (NEUwor), post-traumatic stress disorder (PTSD), and schizophrenia (SCZ) have identified numerous risk loci. 

Genetic influences on psychiatric disorders transcend diagnostic boundaries, indicating that contributing loci confer risk for multiple disorders. Pleiotropy may indicate that over a wide array of biological taxa and traits, the phenotype and its genetic basis could have a modular structure. Strong genetic, phenotypic, and evolutionary relationships exist between some traits, which often share underlying developmental processes or participate in a common functional process [@Pavlicev2015]. To this end, pleiotropy was mapped for 12 psychiatric and neurological disorders by estimating genetic correlations. Genetic correlation is a population parameter that reflects the correlation between the genetic effects on a pair of complex traits or disorders. These disorders were selected based on their common loci and shared biological structures as revealed by earlier studies such as those performed by @Lee2019. In doing so, the extent and biological significance of cross-disorder genetic influences are elucidated [@Smoller2019]. This study aims to show shared and unique genetic bases among disorders characterized by compulsive behaviours (anorexia nervosa); mood and psychotic disorders (schizophrenia, bipolar disorder, and major depression); and early-onset neurodevelopmental disorders combined with major depression (autism spectrum disorder and attention-deficit/hyperactivity disorder). One of these shared genetic bases might be indicated by the existence of genetic liabilities for major depression on two common factors [@Rice2019]. The first of these common factors in relation to bipolar disorder and schizophrenia, the second in relation to attention-deficit/hyperactivity disorder and autism spectrum disorder. This coincides with recent findings showing an association of early-onset depression with genetic risk for attention-deficit/hyperactivity disorder and neurodevelopmental phenotypes [@Rice2019]. Another example of shared genetic bases between disorders was found by @Pasman2018. In their genome wide association study (GWAS), lifetime cannabis use was found to be differentially expressed for cannabis users versus nonusers. Genetic correlations were found between substance use and mental health-related traits, including smoking, alcohol use, schizophrenia, attention-deficit/hyperactivity disorder, and risk-taking behaviour. Furthermore, evidence was found for a causal positive influence of schizophrenia risk on cannabis use [@Pasman2018]. In this perspective, lifetime cannabis use may share genetic bases with schizophrenia and attention-deficit/hyperactivity disorder. Another possible genetic basis might be found when further studying strongly overlapping genetic risk for post-traumatic stress disorder with schizpohrenia, and modest overlap with bipolar disorder and major depression [@Duncan2018]. Identifying individual single-nucleotide polymorphisms (SNPs) exceeding genome-wide significance has proved difficult for post-traumatic stress disorder. However, recently @Nievergelt2019 found ethnicity-specific significant SNPs, two in European and one in African-ancestry analyses. Mapping pleiotropy of these disorders may result in a more thorough image of genetic foundation of these disorders and communalities between disorders regarding these foundations 

Additional possible genetic, phenotypic, and evolutionary relationships existing between traits might be identified when studying the genetic foundation of neuroticism. @Nagel2018 have shown a positive genetic correlation between neuroticism and depression. This study also identified two subclusters of neuroticism; "depressed affect" and "worry". These subclusters differed in their amount of loci shared with depression, suggesting differing causal genetic mechanisms between the subclusters. Furthermore, the correlations for depressed affect and worry with anorexia nervosa, schizophrenia, and ever-smoker differed significantly and sometimes were in opposite direction (e.g. BMI). Loci related to neuroticism were found  in genes expressed in several brain regions. Therefore, any difference between neuroticism subclusters may also be expressed differently in these brain regions. Another example of shared genetic basis is autism spectrum disorder, @Smoller2019 have found most of its associations to co-occur with schizophrenia and bipolar disorders. Some smaller associations were featured for attention-deficit/hyperactivity disorder (16%) and anorexia nervosa (7%). It could thus be expected that stronger genetic relationships are found for autism spectrum disorder with schizophrenia and bipolar disorder than for attention-deficit/hyperactivity disorder and anorexia nervosa. Furthermore, autism spectrum disorder has been found to be comorbid with many disorders. The higher rates of depression in family members of autism spectrum disorder probands compared to the general population has raised the possibility of autism spectrum disorder being related to major depression and bipolar disorder [DeLong, 2004; cited by @Towbin2005]. 

Other reasons to suspect pleiotropy may play a role. In the etiology of psychiatric and neurological disorders is a similarity in terms of phenotypic presentation. Such was the case for Alzheimer’s disease, for which the second of three groups of symptoms comprises of psychiatric symptoms and behavioural disturbances, such as depression, hallucinations, delusions, anxiety, and agitation [@Burns2009]. These symptoms might point to a genetic relation with psychiatric disorders with similar symptoms, such as major depression, anxiety disorder, and schizophrenia. Like Alzheimer’s disease, insomnia may share a genetic base with depression, anxiety, and substance abuse and dependency [@Riemann2007]. The comorbidity between insomnia and psychiatric disorders is well-established and some researchers claim that insomnia isn't simply a symptom of psychiatric disorders but is a risk factor for the development of such conditions [@Riemann2001]. Empirical studies suggest that sleep alterations may not be identical over psychiatric disorders [Philipsen, Feige, Al-Shajlawi, et al., 2005; cited by @Riemann2007]. In this regard, the presence of insomnia was associated with increased risk of depression, anxiety disorders, drug abuse or dependence, and nicotine dependence. Patients with depression demonstrated that, along with disturbances of sleep continuity, sleep is characterized by a reduction of slow wave sleep, disinhibition of REM sleep, and a shortening of REM latency. In addition, the first REM period is prolonged, with increased REM density [Riemann, Berger, & Voderholzer, 2001; cited by @Riemann2007]. In a similar manner, sleep abnormalities are also common features of affective disorders, alcohol dependence, dementia, and schizophrenia [Benca, Obermeyer, Thisted, et al., 1992; cited by @Riemann2007]. Furthermore, one study reported a tendency for shortened REM latency and significantly reduced non-REM sleep in patients with borderline personality disorders [Philipsen, Feige, Al-Shajlawi, et al., 2005; cited by @Riemann2007]. Another study among adults with attention-deficit/hyperactivity disorder showed increased periodic leg movements accompanied by lower subjective quality of sleep compared to healthy controls. Also objective total sleep time in the attention-deficit/hyperactivity disorder patients increased compared to controls [Philipsen, Feige, Hesslinger, et al., 2005; cited by @Riemann2007]. Therefore, the underlying gene expression profiles for insomnia-comorbid psychiatric disorders might not be of one particular nature either. Rather they could differ quite substantially.

Despite the abundance of evidence for the presence of pleiotropy, the exact timing, nature and mechanisms of pleiotropic effects remain unclear [@Lee2019]. Furthermore, identifying “actionable” genes that confer risk for psychiatric disease has proven difficult, despite sizeable studies [@Skene2018]. Therefore, after identifying pleiotropic relations using genetic correlations, we identified genes that are part of the brain-, life phase-, and cell type-specific transcriptome which are afore mentioned disorders. Identifying these genes was done by partitioning heritability through linkage disequilibrium score regression (LDSC). This method relies on the fact that the $\chi$^2^ association statistic for a given SNP includes the effects of all SNPs tagged by this SNP. Thus, for a polygenic trait, SNPs with a high LD score will have higher $\chi$^2^ statistics on average than SNPs with a low LD score [@Bulik2015a]. This phenomenon might be driven either by the higher likelihood of these SNPs tagging an individual large effect or their ability to tag multiple weak effects [@Finucane2015]. If we partition SNPs into functional categories with different contributions to heritability, then LD to a category that is enriched for heritability will increase the $\chi$^2^ statistic of a SNP more than LD to a category that does not contribute to heritability. Hence LDSC estimates the proportion of genome-wide SNP heritability attributable to various functional categories, using information from all SNPs and explicitly modeling LD [@Finucane2015]. Rather than interpreting individual risk genes, connecting genomic results as a whole to cellular studies is crucial since it would allow for prioritization of cells and developmental stages fundamental to the genesis of psychiatric and neurological disorders. Relating life phase-, cell type- and tissue-specific RNA transcriptome expression to psychiatric and neurological disorders could indicate the timing, nature and mechanisms of these pleiotropic effects. These RNA expression profiles were formed by matching the GWAS summary statistics of these disorders to cell type- and developmental period-specific enriched tissue in two brain regions. The brain regions which were studied are the hippocampus, selected for its established relationship to schizophrenia [@Skene2018], and the prefrontal cortex. Both brain regions have established data for cell type-specific enrichment analyses on other phenotypes [@Baselmans2019]. Single-cell RNA-sequencing can be used to derive empirical taxonomies of brain cell types [@Skene2018]. This can connect human genomic findings to specific brain cell types defined by gene expression profiles, highlighting to what specific brain cell types (e.g. neuron, microglia, astrocytes, etc.), genetic findings for disorders fit best. Similarly expression in pre- and postnatal samples could help determine the timing of processes related to psychopathology. For specific brain cell types, a cell type-specific set of genes expressed in 15 cell types were obtained [@Baselmans2019]. For developmental periods, genes predominantly expressed in certain pre- and postnatal developmental periods (early-, middle-, and late prenatal phases, infancy, childhood, adolescence, and adulthood) were collected [@Miller2014]. For SNPs in these developmental period genesets, their enrichment in effect on 12 psychiatric and neurological disorders was computed. Additionally, these life phase-, cell type- and tissue-specific RNA expression profiles of disorders were compared to a RNA expression profile of height (HGHT) as a negative control. This functioned as a filter for any false positive results.

Summarizing, the genetic influences on psychiatric disorders indicate pleiotropy of contributing loci, suggesting unique and overlapping modular structures in phenotypical and genetic bases. This may indicate possible broader domains underlying genetic risk to psychopathology and might give cause for a renewal of classification methods of mental illness. This may be reflected in genetic correlations as well as in life phase-, cell type- and tissue-specific RNA transcriptome expression. The aim of this study is to connect human genomic findings to specific developmental-, brain tissue-, and cell types defined by gene expression profiles. Genes are overexpressed in certain cell types, during a life phase or in a specific brain region. We test where sets of genes associated with a specific cell type or developmental period are differentially enriched in their effect on psychiatric and neurological disorders. In order to achieve this, first it is determined which genetic correlations exist between disorders. Afterwards, cell type- and developmental period-specific expression profiles are constructed for each disorder and compared in order to establish which cell types and developmental periods are differentially enriched in their effect on aforementioned psychiatric and neurological disorders. Height was used as a negative control. We expect genes expressed in certain brain cells or in the brain at certain developmental stages aren't enriched in their effect on height. Any findings are expected to be concordant with relationships established in earlier research findings, whether it be of genetic or phenotypic nature. Hypothetically, these relationships are reflected in developmental period-, brain tissue-, and cell type-specific expression profiles for the neurological and psychiatric disorders. In order to substantiate this, two large data resources have been used. The first large data resource consists of public genome wide association results for the aforementioned psychiatric and neurological disorders. Secondly, the RNA expression profiles of specific brain cell types and developmental periods have been used.

\newpage

# Method

```{r, results='asis'}
table1 <- tibble(
  Disorder = c("ADHD", "ALZ", "AN", "ASD", "BIP", "CAN", "INS", "MD", "NEUdep", "NEUwor", "PTSD", "SCZ", "Height"),
  Cases = c(19099, 71880, 16992, 18381, 20352, 53179, 109389, 130664, NA, NA, 32428, 34241, NA),
  Controls = c(34194, 383378, 55525, 27969, 31358, 131586, 277144, 330470, NA, NA, 174227, 45604, NA),
  'Total samples' = Cases + Controls,
 # 'Case-Control Ratio' = (Cases / Controls) %>% round(2),
  #'Number of GWAS loci' = c(nrow(ADHD), nrow(ALZ), nrow(AN), nrow(ASD), nrow(BIP), nrow(INS), nrow(MDD), nrow(NEU_dep), nrow(NEU_wor), nrow(PTSD), nrow(SCZ), nrow(SUD_can), nrow(HGHT)),
  'GWAS loci' = c(11, 25, 10, 2, 18, 6, 13, 11, 63, 71, 2, 94, 263), 
  'Sample prev.' = (Cases / sum(Cases, na.rm = TRUE)) %>% round(2),
  'Pop. prev. (k)' = c(.05, .003368, rep(.01, times = 3), .43, .1, .15, rep(NA, times = 2), .078, .01, NA),
  'SNP h2' = c(.2213, .0114, .1364, .1119, .2005, .1139, .0593, .1064, .088, .0892, .0258, .2503, .4552)  %>% round(3),
  'se' = c(.0143, .0018, .0089, .0097, .0101, .0071, .0026, .0065, .0034, .0032, .0047, .01, .0193) %>% round(3),
  add_this = rep(")", times = 13),
  References = c("Demontis et al., 2019", "Jansen et al., 2019", "Watson et al., 2019", "Grove et al., 2019", "Stahl et al., 2018", "Pasman et al., 2018", "Jansen et al., 2019", "Wray et al., 2018", rep("Nagel et al., 2018", times = 2), "Nievergelt et al., 2019", "PGC, 2014", "Yengo et al., 2018")
  ) %>%
  unite("SNP h2 (se", c(`SNP h2`, se), sep = " (") %>%
    unite("SNP h2 (se)", c(`SNP h2 (se`, add_this), sep = "")
list_samp_prev_h2_liab <- table1$Cases / (table1$Cases+table1$Controls)

# Neudep, NEUwor, and Height are not disorders and therefore have no cases and controls which can be summed.
# However total N samples of their respective GWASes is known and inserted into the table by use of this for-loop.
for (i in table1$Disorder) {
  if (i == "NEUdep")(
      table1[which(table1$Disorder == i), ]$`Total samples` <- 357957
      )
  if (i == "NEUwor")(
    table1[which(table1$Disorder == i), ]$`Total samples` <- 348219
    )
  if (i == "Height")(
    table1[which(table1$Disorder == i), ]$`Total samples` <- 693529
    )
}

# Adding row with total N for cases, controls and total study samples.
table1 <- rbind(table1,
                c("Total", 
                  sum(table1$Cases, na.rm = TRUE), 
                  sum(table1$Controls, na.rm = TRUE), 
                  sum(table1$`Total samples`), 
                  #(sum(table1$Cases, na.rm = TRUE) / sum(table1$Controls, na.rm = TRUE)) %>% round(2),
                  rep(NA, times = 3), # remove 2 if line above and below are applicable, hence if case:control ratio stays in table
                  #(sum(table1$Cases, na.rm = TRUE) / sum(table1$Cases, na.rm = TRUE)) %>% round(2),
                  rep(NA, times = 3)
                  )
                )
#NAs replaced by empty spaces
table1[is.na(table1)] <- "" 
table1[table1 == "NA (NA)"] <- ""

apa_table(
  table1, 
  caption = "Summary of Twelve Neurological and Psychiatric Disorders, and Height.", 
  escape = FALSE, 
  font_size = "scriptsize",
  span_text_columns = TRUE,
  landscape = FALSE, 
  #placement = "hb",
  #format.args = list(width = NULL)
)

```
## GWAS Data Selection and Cleaning

Genotype data from twelve studies of genetic associations with neurological and psychiatric disorders, and one of height, were included in this report. The twelve neurological and psychiatric disorders included attention-deficit/hyperactivity disorder [@Demontis2019], Alzheimer’s disease [@Jansen2019b], anorexia nervosa [@Watson2019], autism spectrum disorder [@Grove2019], bipolar disorder [@Stahl2019], insomnia [@Jansen2019a], major depression [@Wray2018], neuroticism depressed affect [@Nagel2018], neuroticism worry [@Nagel2018], post-traumatic stress disorder [@Nievergelt2019], schizophrenia [@SWGPGC2014], and lifetime cannabis use [@Pasman2018]. The height data was derived from @Yengo2018. GWAS data were collected from the Psychiatric Genomics Consortium, LD Hub [@Zheng2017], Broad Institute, and CTG Lab websites. The GWAS data consisted of publicly available summary statistics. This resulted in a combined sample of `r table1[14, 2]` cases, and `r table1[14, 3]` controls, however the total number of samples remains higher yet (N = `r table1[14, 4]`, Table 1) because some of the included studies were not of a case-control nature. Neuroticism depressed affect and neuroticism worry had been constructed from ordinal scores on four EPQ-RS items used in its respective study [@Nagel2018]. All study participants were reportedly of European ancestry. 

Standardized and uniform quality control consisted of reformatting summmary statistics using default settings of the "munge_sumstats.py" script in the LDSC software [@Bulik2015a]. These settings filtered SNPs with imputation quality below 0.9, minor allele frequency below 0.01, and strand-ambiguous SNPs. Furthermore, the munged summary statistics were merged with HapMap3 SNP list of European origin (The International HapMap 3 Consortium, 2010). This selected only those SNPs present in populations of European ancestry.

## Liability Scale Heritability

In order to compute the liability scale heritability (Table 1), the GWAS summary statistics of each disorder was matched to a 1000 Genomes European LD score reference panel (The 1000 Genomes Project Consortium, 2015). Afterwards, LD scores with zero variance were removed. Consecutively, the total liability scale heritability was computed (Table 1) for each disorder using open source LD score regression software [@Bulik2015a]. Some of the population prevalences in table 1 should be considered only rough estimates with limited validity, which in turn could affect the liability scale heritability. Population prevalences were based on data provided by @Lee2019 for attention-deficit/hyperactivity disorder, anorexia nervosa, autism spectrum disorder, bipolar disorder, major depression, and schizophrenia. In relation to Alzheimer’s disease population prevalence, @Mayeux2012 state global prevalence of dementia to be estimated as high as 24 million, which was divided by the estimated world population at that time (by populationpyramid.net), of 7,125,827,956. Taking into consideration that this study mentions that the global prevalence of dementia is predicted to double every 20 years until at least 2040, the stability of this estimated population prevalence is debatable. Insomnia population prevalence was based on @Roth2007. post-traumatic stress disorder population prevalence was based on @Kessler1995. They estimated the U.S. life-time prevalence of post-traumatic stress disorder to be 7.8%, based on DSM-III-R criteria and sampled from 5,877 persons aged 15 to 54 years. 10 years later, @Kessler2005 estimated life-time prevalence to be 6.8% based on DSM-IV criteria and sampled from 9,282 persons aged 18 years or older. Several variables differ in their exact definitions when comparing these studies. Furthermore, both studies were conducted among inhabitants of the U.S. Therefore it is assumed that U.S. statistics are broadly indicative for post-traumatic stress disorder prevalence worldwide. This may not be true since the World Health Organization had collected data from nearly 200,000 respondents in 27 countries as of 2008 [@Direk2010]. Their estimates for lifetime post-traumatic stress disorder prevalence range from 0.3% in China to 6.1% in New Zealand. Also, statistics reported from these various countries are not directly comparable due to methodological differences in survey administration and sampling strategies. This complicates estimating the population prevalence of post-traumatic stress disorder. In relation to lifetime cannabis use, among Canadians aged 15 years and older, an estimated 43% have used cannabis in their lifetime [@Hosseini2019]. This number is probably highly variable across countries and may depend on (historic) policies regarding cannabis use. Therefore the estimated prevaluence might not be representative of population prevalence worldwide. For heritability, it is customary to report heritability on the liability scale, because liability scale heritability is comparable across studies with different prevalences. However neuroticism depressed affect and neuroticism worry were both constructed from ordinal scores on four EPQ-RS items of the UKB cohort used in its study. Therefore they weren't of a case-control nature and lacked information regarding their prevalences. Hence calculating corresponding heritability on a liability scale was not feasible. Instead, their observed scale heritability was added in Table 1. Height served as a control measurement and wasn't of a case-control nature either. The use of an observed scale heritability rather than a liability scale was applicable here as well.

## Genetic Correlation

This study used GWAS summary statistics in order to estimate genetic correlations. The correlations are obtained by regressing the product of association test statistics for a pair of traits at a given SNP on the LD score for the given SNP. This LD score of a SNP is the sum of LD $r^2$ measured with all other SNPs and can be calculated in a reference sample of the same ethnicity. Practical estimation is done using LDSC [@Bulik2015b], which is a tool for analysis of GWAS summary statistics which requires lower computing power and time as compared to other methods such as genomic restricted maximum likelihood [@Ni2018]. LDSC assumes that the GWAS sample has been drawn from the same ethnic population as the reference sample used to calculate the LD scores and exploits the relationship between association test statistic and LD score as expected under polygenicity. Genetic correlations were considered statistically significant at a Bonferroni correction for multiple testing of *P* < `r sig_level_corrs %>% formatC(format = "e", digits = 2)`.

## Partitioned SNP-Heritability

LDSC was used in order to determine whether genomic regions are enriched for genetic effects on the neurological and psychiatric disorders. Enrichment of a category was defined to be the proportion of SNP heritability in the category divided by the proportion of SNPs in that category [@Finucane2015]. Following the procedure of @Finucane2015, LDSC was estimated for the baseline model, which contained 53 categories. These categories included coding, UTR, promoter and intronic regions, the histone marks monomethylation (H3K4me1) and trimethlation (H3K4me3) of histone H3 at lysine 4, acetylation of histone H3 at lysine 9 and two versions of acetylation of histone H3 at lysine 27 (H3K27ac), open chromatin as reflected by DNase I hypersensitivity sites (DHSs), combined chromHMM and Segway predictions, super-enhancers, and enhancers with balanced bidirectional capped transcripts. Windows of 500 basepairs around each of these annotations were included in order to prevent estimates from being biased upwards. Additionally, analyses were performed by using cell type-specific gene-expression annotations, for the cell type-specific analyses, or developmental period-specific gene-expression annotations, for the developmental period-specific analyses. 

Data for cell type-specific analyses were collected from @Baselmans2019, who had obtained a publicly available matrix of gene counts generated on the basis of single nuclei from the prefrontal cortex and hippocampus of human donors by @Habib2017. @Baselmans2019 divided these nuclei into 15 cell types: astrocytes (ASC1 and ASC2), endothelial cells (END), pyramidal neurons from the hippocampal CA region (exCA1 and exCA3), granule neurons from the hippocampal dentate gyrus region (exDG), glutamatergic neurons from the prefrontal cortex (exPFC1 and exPFC2), GABAergic interneurons (GABA1 and GABA2), microglia (MG), oligodendrocyte 1 (ODC1), oligodendrocyte precursors cells (OPC), neuronal stem cells (NSC), and Unclassified cells; and computed cell type-specific gene expression for the different types of brain cells. Based on this geneset, @Baselmans2019 computed partitioned LD scores.

Data for developmental period-specific analyses were collected from the BrainSpan Atlas of the Developing Human Brain [@Miller2014] from whence RNA-Seq Gencode v10 summarized to genes was used. This matrix of summarized gene counts was divided into seven groups based on their developmental periods: early prenatal (8 - 12 pcw), middle prenatal (13 - 24 pcw), late prenatal (25 - 38 pcw), infancy (birth - 18 months), childhood (19 months - 11 years), adolescence (12 - 19 years), and adulthood (20 - 60+ years). These seven developmental groups were compressed out of BrainSpan Atlas' original 13 groups because some groups contained little to no data. This could result in loss of signal, however not providing sufficient amounts of data in turn could result in lack of statistical power to detect any genetic effects. Annotations were made based on the differential gene expression for each of these developmental period as compared to all other developmental period and brain areas (global differential gene expression), based on Love, Huber, and Anders (2014). Due to the nature of the RNA-Seq Gencode data, normalized counts were used. This is not ideal for the DESeq2 model as this model corrects internally for library size [@R-DESeq2]. The software used for making the annotations was publicly available open source software written in R (github.com/hillfung/make_LDSR_annot). Subsequently, LD scores were computed based on these annotations. 

A false discovery rate (FDR) of < 5% was considered for significance. The FDR was calculated using the Benjamini-Hochberg method [@Benjamini1995]. In LDSC network plots, three different colours are visible based on their FDR significance thresholds: green for FDR < 0.05, yellow for FDR < 0.01, and pink for FDR < 0.001. Yellow squares were indicative of either cell types or developmental period, whereas disorders each had their unique colouring. In addition, the enrichment of genes differentially expressed in cell type-, brain tissue-, and developmental period-specific expression profiles on height was used as a negative control [@Wood2014].

## Software

Analyses were performed using Unix shell scripts on the Lisa Compute Cluster of SURFsara with the modules "2019", and "Anaconda3". In most cases Python3 was used with dependencies bitarray (0.8.3), nose (1.3.7), pybedtools (0.8.0), scipy (1.2.1), pandas (0.24.2), and numpy (1.18.1). However, the LDSC software had some compatibility issues for Python3, hence incidental switches were made to Python2 with different version for pandas (0.18.1) and numpy (1.15.4).

LDSC software was provided by @Bulik2015a and publicly available open source software was used to make a *r~g~* matrix (github.com/hillfung/rg-matrix) and LDSC network plots [@Kanai2018]. LDSC network plots were edited further using Paint 3D.

A wide array of R packages has been used for plotting figures, calculating the number of significant GWAS loci per disorder, and performing analyses of developmental period- and cell type-specific enrichment, differential expression of developmental periods, and making annotation files for these developmental periods. For this, we made use of `r cite_r("r-references.bib")`.

For a full overview of the Unix shell- and R-scripts used, see github.com/timfinke/internship1.

\newpage

# Results
```{r }
table_rg <- read.table(file = file.path(path_data_rg_matrix, "rg.tab"))[,-1] %>% as.matrix()
table_rg_p <- read.table(file = file.path(path_data_rg_matrix, "rg_p.tab"))[,-1] %>% as.matrix()
table_rg_p2 <- table_rg_p # not adapted by formatC()
table_rg_se <- read.table(file = file.path(path_data_rg_matrix, "rg_se.tab"))[,-1] %>% as.matrix()
table_cti <- read.table(file = file.path(path_data_rg_matrix, "cti.tab"))[,-1] %>% as.matrix()
table_cti_se <- read.table(file = file.path(path_data_rg_matrix, "cti_se.tab"))[,-1] %>% as.matrix()

colnames(table_rg) = rownames(table_rg) = colnames(table_rg_p) = rownames(table_rg_p) = colnames(table_rg_se) = rownames(table_rg_se) = colnames(table_cti) = rownames(table_cti) = colnames(table_cti_se) = rownames(table_cti_se) = c("ADHD", "ALZ", "AN", "ASD", "BIP", "INS", "MD", "NEUdep", "NEUwor", "PTSD", "SCZ", "CAN")

table_rg_only_sig <- ifelse(table_rg_p < sig_level_corrs, 1, 0)
table_rg_only_sig <- table_rg * table_rg_only_sig
table_rg_p <- table_rg_p %>% formatC(format = "e", digits = 2)
results_sig_rg <- table_results %>%
  filter(p < sig_level_corrs)
```
## Genetic Correlations among Disorders 

Genetic correlations were estimated using LDSC. The pairwise analysis of the summary statistics of 12 psychiatric and neurological disorders resulted into `r nrow(results_sig_rg)` statistically significant genetic correlations. The significant genetic correlations (P < `r sig_level_corrs %>% formatC(format = "e", digits = 2)`) are visualized in Figure 1 with the use of asterisks. Blue dots indicate positive correlations whereas red dots indicate negative correlations. 

```{r figure 1}
corrplot(table_rg, type = "upper", is.corr = TRUE, diag = FALSE, 
                  tl.col = "black", cl.lim = c(-1, 1), p.mat = table_rg_p2,
                  insig = "label_sig", sig.level = c(sig_level_corrs), pch.cex = 1, # sig.level = c(5e-8, 1e-6, sig_level_corrs)
                  title = "", tl.srt = 45, mar=c(0,0,1,0))
```
*Figure 1.* Genetic correlation matrix. Asterisks indicate significant genetic correlations (*P* < `r sig_level_corrs %>% formatC(format = "e", digits = 2)`). Positive relations are indicated by blue dots, negative relations by red dots. Large circle sizes correspond to more significant P values.

The strongest correlations were found between major depression and post-traumatic stress disorder (*r~g~* = `r table_rg[7,10]`, se = `r table_rg_se[7,10]`, *P* = `r table_rg_p[7,10]`), Major depression and neuroticism depressed affect (*r~g~* = `r table_rg[7,8]`, se = `r table_rg_se[7,8]`, *P* = `r table_rg_p[7,8]`), and neuroticism depressed affect and post-traumatic stress disorder (*r~g~* = `r table_rg[8,10]`, se = `r table_rg_se[8,10]`, *P* = `r table_rg_p[8,10]`). 

A slightly weaker relationship was found between neuroticism worry and post-traumatic stress disorder (*r~g~* = `r table_rg[9,10]`, se = `r table_rg_se[9,10]`, *P* = `r table_rg_p[9,10]`) and major depression (*r~g~* = `r table_rg[9,7]`, se = `r table_rg_se[9,7]`, *P* = `r table_rg_p[9,7]`). As well as a strong relationship between neuroticism depressed affect and neuroticism worry (*r~g~* = `r table_rg[9,8]`, se = `r table_rg_se[9,8]`, *P* = `r table_rg_p[9,8]`). In addition, some of the differential relationships for neuroticism depressed affect and neuroticism worry with other disorders have been replicated. @Nagel2018 had found relationships for neuroticism depressed affect with attention-deficit/hyperactivity disorder and for neuroticism worry with anorexia nervosa, bipolar disorder, and schizophrenia. Similar results were obtained in this study. A newly found result is however the positive relationship between neuroticism depressed affect and anorexia nervosa (*r~g~* = `r table_rg[8,3]`, se = `r table_rg_se[8,3]`, *P* = `r table_rg_p[8,3]`). This relationship is weaker than the relationship between neuroticism worry and anorexia nervosa (*r~g~* = `r table_rg[9,3]`, se = `r table_rg_se[9,3]`, *P* = `r table_rg_p[9,3]`). Nonetheless this suggests that neuroticism worry is not unique as a neuroticism subcluster in its relationship with anorexia nervosa.

Overall, results found were largely concordant with those of @Lee2019, with some new insights due to the use of previously unused disorder summary statistics. However, some discordant results were found as well. No significant relationship was found between anorexia nervosa and attention-deficit/hyperactivity disorder (*r~g~* = `r table_rg[1,3]`, se = `r table_rg_se[1,3]`, *P* = `r table_rg_p[1,3]`), whereas @Lee2019 did find a negative genetic association (*r~g~* = -0.26, *P* < 0.0001). These differences are most likely attributable to the use of GWAS studies with higher numbers of participants in this study (N of `r table1[3,4]` vs. 14478).

Regarding found genetic correlations and the degree to which they were concordant with expectations formulated earlier. A moderate genetic correlation was found for lifetime cannabis use and schizophrenia (*r~g~* = `r table_rg[12,11]`, se = `r table_rg_se[12,11]`, *P* = `r table_rg_p[12,11]`). This lends validity to the posibility of a positive relationship between genetic risk factors of schizophrenia on lifetime cannabis use. Furthermore, significant genetic correlations were found for post-traumatic stress disorder with schizophrenia (*r~g~* = `r table_rg[10,11]`, se = `r table_rg_se[10,11]`, *P* = `r table_rg_p[10,11]`) and major depression (*r~g~* = `r table_rg[10,7]`, se = `r table_rg_se[10,7]`, *P* = `r table_rg_p[10,7]`). Earlier research suggested strong overlapping genetic risk for post-traumatic stress disorder with schizophrenia and moderate overlapping genetic risk with major depression [@Duncan2018]. However these genetic correlations seem to suggest a stronger genetic relationship of post-traumatic stress disorder with major depression than with schizophrenia. Regarding lifetime cannabis use, other than the afore mentioned genetic correlation with schizophrenia, relationships were found with attention-deficit/hyperactivity disorder (*r~g~* = `r table_rg[12,1]`, se = `r table_rg_se[12,1]`, *P* = `r table_rg_p[12,1]`), anorexia nervosa (*r~g~* = `r table_rg[12,3]`, se = `r table_rg_se[12,3]`, *P* = `r table_rg_p[12,3]`), autism spectrum disorder (*r~g~* = `r table_rg[12,4]`, se = `r table_rg_se[12,4]`, *P* = `r table_rg_p[12,4]`), bipolar disorder (*r~g~* = `r table_rg[12,5]`, se = `r table_rg_se[12,5]`, *P* = `r table_rg_p[12,5]`), major depression (*r~g~* = `r table_rg[12,7]`, se = `r table_rg_se[12,7]`, *P* = `r table_rg_p[12,7]`), and neuroticism worry (*r~g~* = `r table_rg[12,9]`, se = `r table_rg_se[12,9]`, *P* = `r table_rg_p[12,9]`). These genetic correlations highlight the possible role lifetime cannabis use may play in the development of these disorders and merits further study of the underlying relationships.

Some expected relationships failed to be replicated in the genetic correlations. Autism spectrum disorder did not show any significant genetic correlations with bipolar disorder or anorexia nervosa. However relationships which did fulfil expectations were found for autism spectrum disorder with schizophrenia (*r~g~* = `r table_rg[4,11]`, se = `r table_rg_se[4,11]`, *P* = `r table_rg_p[4,11]`), major depression (*r~g~* = `r table_rg[4,7]`, se = `r table_rg_se[4,7]`, *P* = `r table_rg_p[4,7]`) and attention-deficit/hyperactivity disorder (*r~g~* = `r table_rg[4,1]`, se = `r table_rg_se[4,1]`, *P* = `r table_rg_p[4,1]`). Insomnia did not show the expected genetic relationship with lifetime cannabis use. However genetic correlations of insomnia with attention-deficit/hyperactivity disorder (*r~g~* = `r table_rg[6,1]`, se = `r table_rg_se[6,1]`, *P* = `r table_rg_p[6,1]`), major depression (*r~g~* = `r table_rg[6,7]`, se = `r table_rg_se[6,7]`, *P* = `r table_rg_p[6,7]`), neuroticism depressed affect (*r~g~* = `r table_rg[6,8]`, se = `r table_rg_se[6,8]`, *P* = `r table_rg_p[6,8]`), neuroticism worry (*r~g~* = `r table_rg[6,9]`, se = `r table_rg_se[6,9]`, *P* = `r table_rg_p[6,9]`), and post-traumatic stress disorder (*r~g~* = `r table_rg[6,10]`, se = `r table_rg_se[6,10]`, *P* = `r table_rg_p[6,10]`) were found. Alzheimer's disease neglected to result in any significant genetic correlations altogether.

All genetic relations are further visualized for correlations larger than 0.1 in Figure 2. In this figure, wide edges and dark colours indicate strong relationships, whereas small edges and light colours indicate weaker relationships. The exact *r~g~* values between disorders are presented in the figure.

```{r  figure 2}
qgraph(table_rg_only_sig, shape = "circle", posCol = "blue", #graph = "cor"
       negCol = "red", layout = "spring", threshold = 0.1,
       edge.labels = TRUE, edge.label.cex = .8, edge.label.bg = FALSE, edge.label.color = "black",
       edge.label.position = .67, labels = colnames(table_rg_only_sig), label.scale.equal = TRUE, label.scale = FALSE,
       label.cex = .6, curveAll = FALSE, repulsion = .3, title = "", title.cex = 1.2)
```
*Figure 2.* Genetic correlation in-directed graph. All significant correlations > 0.10 are portrayed. Wide edges and dark colours indicate strong relationships, small edges and light colours indicate weak relationships. Exact *r~g~* values between disorders are presented.

```{r}
AN_UN <- input_ct %>% 
  filter(trait == "AN") %>%
  filter(Category == "Unclassified")

MD_GABA1 <- input_ct %>% 
  filter(trait == "MDD") %>%
  filter(Category == "GABA1")
MD_GABA2 <- input_ct %>% 
  filter(trait == "MDD") %>%
  filter(Category == "GABA2")
MD_MG <- input_ct %>% 
  filter(trait == "MDD") %>%
  filter(Category == "MG")
Enr_MD_MG <- input_ct_enrichment %>% 
  filter(trait == "MDD") %>%
  filter(Category == "MG")

# differently enriched NEUdep and NEUwor
Enr_NEUdep_END <- input_ct_enrichment %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "END")
Enr_NEUdep_MG <- input_ct_enrichment %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "MG")
Enr_NEUdep_ODC1 <- input_ct_enrichment %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "ODC1")

Enr_NEUwor_exCA3 <- input_ct_enrichment %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "exCA3")
Enr_NEUwor_exPFC1 <- input_ct_enrichment %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "exPFC1")

# similar enriched NEUdep and NEUwor
Enr_NEUdep_GABA2 <- input_ct_enrichment %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "GABA2")
Enr_NEUwor_GABA2 <- input_ct_enrichment %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "GABA2")

Enr_NEUdep_exCA1 <- input_ct_enrichment %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "exCA1")
Enr_NEUwor_exCA1 <- input_ct_enrichment %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "exCA1")

# similar coefficients NEUdep and NEUwor for GABA2
NEUdep_GABA2 <- input_ct %>% 
  filter(trait == "NEUdep") %>%
  filter(Category == "GABA2")
NEUwor_GABA2 <- input_ct %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "GABA2")

# unique coefficient NEUwor for exPFC1
NEUwor_exPFC1 <- input_ct %>% 
  filter(trait == "NEUwor") %>%
  filter(Category == "exPFC1")

# coefficients SCZ
SCZ_exPFC1 <- input_ct %>% 
  filter(trait == "SCZ") %>%
  filter(Category == "exPFC1")
SCZ_GABA2 <- input_ct %>% 
  filter(trait == "SCZ") %>%
  filter(Category == "GABA2")


```

\newpage

## Partitioned SNP-Heritability

\begin{figure}[htbp]
\centering
\includegraphics{./images/network_ct_plot_af.png}
\caption{LDSC network for cell type-specific enrichment of disorders. FDR thresholds are indicated by green (< 0.05), yellow (< 0.01), and pink (< 0.001) lines. Thicker edges correspond to more significant FDRs.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{./images/network_ct_coef_af.png}
\caption{LDSC network for cell type-specific enrichment coefficients of disorders. FDR thresholds are indicated by green (< 0.05), yellow (< 0.01), and pink (< 0.001) lines. Thicker edges correspond to more significant FDRs.}
\end{figure}

\newpage

### Cell type-specific enrichment

The LDSC network plots show the strength of shared genetic relationships between disorders and the cell types for which these shared bases are of influence. When studying the enrichment levels, a clear network of relations is shown with significant overlap in originating cell types between several disorders (Figure 3). When looking for a directional effect of these enrichments, significant enrichment coefficients arose for exPFC1 (on neuroticism worry and schizophrenia), MG (on major depression), GABA2 (on neuroticism depressed affect, neuroticism worry, and schizophrenia), and Unclassfied cell types (on anorexia nervosa) (Figure 4). Genes expressed in all cell types except for exPFC1 showed significant enrichment in their effect on height, however no significant enrichment coefficient was found for any of those cell types. Figures 5-9 show enrichment levels and -coefficients for specified disorders on the left and enrichment levels for height on the right. In these figures, a blue dot indicates a significant value at FDR < 0.05. A red dot on the other hand, indicates a value which failes to meet this threshold. Figures showing enrichment levels and -coefficients were presented in this section based on the presence of significant enrichment coefficients for these disorders. Similar figures for all 12 disorders can be found in Appendix I.

```{r figure 5}
plot_celltype("AN", " ")

```
*Figure 5.* Cell type-specific enrichment levels and -coefficients of anorexia nervosa. The figures on the left portray enrichment levels, and -coefficients for anorexia nervosa. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

Regardless of lacking statistical significant enrichment of Unclassified cell types on enrichment levels, a significant positive enrichment coefficient shows for this cell type on anorexia nervosa (coefficient = `r formatC(AN_UN$Coefficient, format = "e", digits = 2)`, se = `r formatC(AN_UN$Coefficient_std_error, format = "e", digits = 2)`, *P* = `r formatC(AN_UN$Coefficient_p, format = "e", digits = 2)`). However the enrichment level for this cell type shows no significant result (Figure 5). Surprisingly, the enrichment level of the Unclassified cell type which, even though not statistically significant, is enriched in a larger amount than other cell types. A further subdivision of this cell type may uncover seperate underlying cell types, of which one may both have a significant enrichment coefficient and be present in abundance. This presence might be masked by the inclusion of other cell types in this group with lower enrichment levels while not being able to dilute the strong signal this cell type emits.
```{r figure 6}
plot_celltype("MDD", " ")

```
*Figure 6.* Cell type-specific enrichment levels and -coefficients of major depression. The figures on the left portray enrichment levels, and -coefficients for major depression. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

For major depression, genes expressed in GABA1, GABA2, and MG showed significant enrichment levels (Figure 6). However statistically significant enrichment coefficients were lacking for GABA1 (coefficient = `r MD_GABA1$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r MD_GABA1$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r MD_GABA1$Coefficient_p %>% formatC(format = "e", digits = 2)`) and GABA2 (coefficient = `r MD_GABA2$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r MD_GABA2$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r MD_GABA2$Coefficient_p %>% formatC(format = "e", digits = 2)`). Taking into account that all but one cell types show significant enrichment levels for height, the relevance of these significant enrichments lacking a significant enrichment coefficient should be called into question. However the enrichment levels of disorders do differ from those of height in some cases. In contrast, a significant negative enrichment coefficient was found for MG on major depression (coefficient = `r MD_MG$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r MD_MG$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r MD_MG$Coefficient_p %>% formatC(format = "e", digits = 2)`), while its enrichment level does show depletion (enrichment = `r Enr_MD_MG$Enrichment`, se = `r Enr_MD_MG$Enrichment_std_error`, *P* = `r Enr_MD_MG$Enrichment_p %>% formatC(format = "e", digits = 2)`). The depleted enrichment level suggests there is less heritability for MG on major depression than can be expected based on the size of the annotation. This may shine a light on the as of yet ambiguous role of MG on major depression in earlier research [@Perry2018].

```{r figure 7}
plot_celltype("NEUdep", " ")

```
*Figure 7.* Cell type-specific enrichment levels and -coefficients of neuroticism depressed affect. The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure 8}
plot_celltype("NEUwor", " ")

```
*Figure 8.* Cell type-specific enrichment levels and -coefficients of neuroticism worry. The figures on the left portray enrichment levels, and -coefficients for neuroticism worry. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

In terms of enrichment levels, further evidence is generated for the existence of genetic differences between the two neuroticism subclusters (Figures 7 and 8). Whereas END (enrichment = `r Enr_NEUdep_END$Enrichment`, se = `r Enr_NEUdep_END$Enrichment_std_error`, *P* = `r Enr_NEUdep_END$Enrichment_p %>% formatC(format = "e", digits = 2)`), MG (enrichment = `r Enr_NEUdep_MG$Enrichment`, se = `r Enr_NEUdep_MG$Enrichment_std_error`, *P* = `r Enr_NEUdep_MG$Enrichment_p %>% formatC(format = "e", digits = 2)`), and ODC1 (enrichment = `r Enr_NEUdep_ODC1$Enrichment`, se = `r Enr_NEUdep_ODC1$Enrichment_std_error`, *P* = `r Enr_NEUdep_ODC1$Enrichment_p %>% formatC(format = "e", digits = 2)`) show depleted enrichment levels in their effect on neuroticism depressed affect, significant enrichment levels were lacking for these cell types in their effect on neuroticism worry. However exCA3 (enrichment = `r Enr_NEUwor_exCA3$Enrichment`, se = `r Enr_NEUwor_exCA3$Enrichment_std_error`, *P* = `r Enr_NEUwor_exCA3$Enrichment_p %>% formatC(format = "e", digits = 2)`) and exPFC1 (enrichment = `r Enr_NEUwor_exPFC1$Enrichment`, se = `r Enr_NEUwor_exPFC1$Enrichment_std_error`, *P* = `r Enr_NEUwor_exPFC1$Enrichment_p %>% formatC(format = "e", digits = 2)`) do show subcluster-specific enrichment in their effect on neuroticism worry. Some common genetic foundation is found as well. Genes expressed in exCA1 (neuroticism depressed affect: enrichment = `r Enr_NEUdep_exCA1$Enrichment`, se = `r Enr_NEUdep_exCA1$Enrichment_std_error`, *P* = `r Enr_NEUdep_exCA1$Enrichment_p %>% formatC(format = "e", digits = 2)`; neuroticism worry: enrichment = `r Enr_NEUwor_exCA1$Enrichment`, se = `r Enr_NEUwor_exCA1$Enrichment_std_error`, *P* = `r Enr_NEUwor_exCA1$Enrichment_p %>% formatC(format = "e", digits = 2)`) and GABA2 (neuroticism depressed affect: enrichment = `r Enr_NEUdep_GABA2$Enrichment`, se = `r Enr_NEUdep_GABA2$Enrichment_std_error`, *P* = `r Enr_NEUdep_GABA2$Enrichment_p %>% formatC(format = "e", digits = 2)`; neuroticism worry: enrichment = `r Enr_NEUwor_GABA2$Enrichment`, se = `r Enr_NEUwor_GABA2$Enrichment_std_error`, *P* = `r Enr_NEUwor_GABA2$Enrichment_p %>% formatC(format = "e", digits = 2)`) show roughly similar enrichment levels in their effects on both neuroticism subclusters. Regarding the enrichment coefficients, these results paint a somewhat similar picture when looking at GABA2 for neuroticism depressed affect (coefficient = `r NEUdep_GABA2$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r NEUdep_GABA2$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r NEUdep_GABA2$Coefficient_p %>% formatC(format = "e", digits = 2)`) and neuroticism worry (coefficient = `r NEUwor_GABA2$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r NEUwor_GABA2$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r NEUwor_GABA2$Coefficient_p %>% formatC(format = "e", digits = 2)`). However exPFC1 does show a significant effect for neuroticism worry (coefficient = `r NEUwor_exPFC1$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r NEUwor_exPFC1$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r NEUwor_exPFC1$Coefficient_p %>% formatC(format = "e", digits = 2)`), where it does not for neuroticism depressed affect.
When comparing both neuroticism depressed affect and neuroticism worry to major depression, it is evident that genes expressed in GABA2 show high levels of enrichment for all three phenotypes. Furthermore genes expressed in MG show depleted enrichment levels for both neuroticism depressed affect and major depression (neurotcisim depressed affect: enrichment = `r Enr_NEUdep_MG$Enrichment`, se = `r Enr_NEUdep_MG$Enrichment_std_error`, *P* = `r Enr_NEUdep_MG$Enrichment_p %>% formatC(format = "e", digits = 2)`; major depression: enrichment = `r Enr_MD_MG$Enrichment`, se = `r Enr_MD_MG$Enrichment_std_error`, *P* = `r Enr_MD_MG$Enrichment_p %>% formatC(format = "e", digits = 2)`), whereas MG were not significantly enriched for neuroticism worry. These results may indicate that enrichment of MG plays a role in the differences in genetic correlation foundation between major depression and neuroticism depressed affect (*r~g~* = `r table_rg[7,8]`, se = `r table_rg_se[7,8]`, *P* = `r table_rg_p[7,8]`) as opposed to that between major depression and neuroticism worry (*r~g~* = `r table_rg[9,7]`, se = `r table_rg_se[9,7]`, *P* = `r table_rg_p[9,7]`).
```{r figure 9}
plot_celltype("SCZ", "")

```
*Figure 9.* Cell type-specific enrichment levels and -coefficients of schizophrenia. The figures on the left portray enrichment levels, and -coefficients for schizophrenia. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

Genes expressed in ExCA1, exCA3, exDG, exPFC1, exPFC2, GABA1, GABA2, OPC and Unclassified cell types all show significant enrichments regarding schizophrenia (Figure 9). However only exPFC1 (coefficient = `r SCZ_exPFC1$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r SCZ_exPFC1$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r SCZ_exPFC1$Coefficient_p %>% formatC(format = "e", digits = 2)`) and GABA2 (coefficient = `r SCZ_GABA2$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r SCZ_GABA2$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r SCZ_GABA2$Coefficient_p %>% formatC(format = "e", digits = 2)`) turn out to have a statistically significant coefficient as well.
Regarding the genetic association between schizophrenia and neuroticism worry  (*r~g~* = `r table_rg[9,11]`, se = `r table_rg_se[9,11]`, *P* = `r table_rg_p[9,11]`), it is indicative that exPFC1 and GABA2 have positive effects on both disorders. These cell type expression profiles may lie at the root of the common genetic basis between the two disorders.
```{r figure 10}
ct_enrichment_HGHT_control1 <- control_for_height(input_ct_enrichment, Nse = 1)

ct_height <- input_ct_enrichment %>% 
  filter(trait == "HGHT")
ct_height2 <- left_join(ct_enrichment_HGHT_control1, ct_height, by = "Category")[,c(1, 21:29, 11:20)]

ct_height2$Legend <- "Height - control"
ct_enrichment_HGHT_control1$Legend <- ct_enrichment_HGHT_control1$trait
ct_enrichment_HGHT_control1$Legend <- str_replace_all(ct_enrichment_HGHT_control1$Legend, 
                                              c("ALZ" = "Alzheimer's disease", 
                                                "NEUdep" = "Neuroticism depressed affect", 
                                                "SCZ" = "Schizophrenia"))

colnames(ct_height2) <- colnames(ct_enrichment_HGHT_control1)
selection <- rbind(ct_enrichment_HGHT_control1, ct_height2)
remove(ct_height, ct_height2)

colours <- c(
    "Height - control" = "yellow",
    "Alzheimer's disease" = "green",
    "Neuroticism depressed affect" = "blue",
    "Schizophrenia" = "red")

selection %>%
  unite("Disorder - Cell type", c(trait, cell_type_clean), sep = " - ") %>%
  ggplot(mapping = aes(`Disorder - Cell type`, Enrichment, group = 1)) +
    geom_point(alpha = 1, 
               size = 3,
               aes(color = Legend),
               show.legend = TRUE) +
    geom_errorbar(aes(ymin = Enrichment - Enrichment_std_error, 
                      ymax = Enrichment + Enrichment_std_error), 
                  width = .25,
                  alpha = .4) +
  scale_colour_manual(
      values = colours,
      limits = names(colours)) +
    ggtitle(" ") +
    theme(plot.title = element_text(size = 10, 
                                    face="bold", 
                                    margin = margin(10, 0, 10, 0))) +
    labs(x = "Disorder per cell type", 
         y = "Enrichment") +
    theme(axis.text.x = element_text(angle = 50, vjust = 1, hjust = 1, size = 10)) +
    theme(panel.background = element_rect(fill = 'Grey90')) +
  theme(legend.text = element_text(size = 8))

```
*Figure 10.* Cell type-specific enrichment levels of disorders differing from cell type-specific enrichment levels of height. Colours are indicative of specified disorders.

In addition to anorexia nervosa, major depression, neuroticism depressed affect, neuroticism worry, and schizophrenia, some significantly enriched cell types were also found for Alzheimer's disease (ASC2, END, exCA1, exDG, exPFC1, MG, NSC, ODC1, Unclassified; Figure S2), bipolar disorder (exCA1, exCA3, exDG, exPFC1, GABA1, GABA2, OPC, Unclassified; Figure S5), and insomnia (GABA2; Figure S7). However for these last three disorders no significant enrichment coefficients for a specific cell type were found (see Supplementary Figures in Appendix I). The genes expressed in these cell types are therefore ambiguous as to the presence of a true effect on these disorders. In order to estimate which of these cell type-specific enriched disorders might have an enrichment coefficient which failed to register, enrichment levels for height were used as a negative control. This resulted in a somewhat different picture for cell type-specific enrichment levels of disorders. Of the disorders that failed to generate a significant enrichment coefficient, six cell types did not have an overlap of the enrichment level standard error with that of height (Figure 10). Four of those were found for Alzheimer's disease. These were END (enrichment = `r round(ct_enrichment_HGHT_control1[1,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[1,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[1,7]`), exCA1 (enrichment = `r round(ct_enrichment_HGHT_control1[2,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[2,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[2,7]`), exDG (enrichment = `r round(ct_enrichment_HGHT_control1[3,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[3,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[3,7]`), and Unclassified (enrichment = `r round(ct_enrichment_HGHT_control1[4,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[4,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[4,7]`). In addition, MG showed no overlap with height for neuroticism depressed affect (enrichment = `r round(ct_enrichment_HGHT_control1[5,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[5,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[5,7]`). Neither did Unclassified for schizophrenia (enrichment = `r round(ct_enrichment_HGHT_control1[6,5], 3)`, se = `r round(ct_enrichment_HGHT_control1[6,6], 3)`, *P* = `r ct_enrichment_HGHT_control1[6,7]`). However it should be noted that these results stem merely from examining overlap, or lack thereof, between the standard error-intervals around cell type-specific enrichment levels of disorders and their corresponding enrichment levels of height. When standard errors do not overlap, there is no assurance that the difference between these enrichment levels is statistically significant. Nonetheless, the genes expressed in the cell types which qualify for this condition, might be interesting candidates for further in-depth study relating to the presence of a true effect on relevant disorders.

```{r }
SCZ_ADO_HIP <- input_age %>% 
  filter(trait == "SCZ") %>%
  filter(Category == "Adolescence") %>%
  filter(trait_category == "Hippocampus")
```

\newpage

\begin{figure}[htbp]
\centering
\includegraphics{./images/network_age_plot_af.png}
\caption{LDSC network for developmental period-specific enrichment of disorders. FDR thresholds are indicated by green (< 0.05), yellow (< 0.01), and pink (< 0.001) lines. Thicker edges correspond to more significant FDRs.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{./images/network_age_coef_af.png}
\caption{LDSC network for developmental period-specific enrichment coefficients of disorders. FDR thresholds are indicated by green (< 0.05), yellow (< 0.01), and pink (< 0.001) lines. Thicker edges correspond to more significant FDRs.}
\end{figure}

\newpage

### Developmental period-specific enrichment

The LDSC network plots show the strength of shared genetic relationships between disorders and the developmental periods at which these shared bases are of influence. When studying the enrichment levels, a clear network of relations is shown with significant overlap in originating developmental periods between several disorders (Figure 11). However when looking for a statistical significant coefficient of these enrichments, only a relationship between adolescence and hippocampal schizophrenia remains (Figure 12). Height showed significant enrichment for all developmental periods, however no significant enrichment coefficient was found. Figures 13-16 show enrichment levels and -coeffficients for specified disorders on the left and enrichment values for height on the right. In these figures, a blue dot indicates a significant value at FDR < 0.05. A red dot on the other hand, indicates a value which failes to meet this threshold. The blue line indicates the linear regression of the enrichment levels or -coefficients over age. Figures showing enrichment levels and -coefficients were presented in this section based on the presence of significant enrichment coefficients for these disorders. Similar figures for all 12 disorders can be found in Appendix II.

```{r  figure 13}
plot_agegroups("SCZ", "Hippocampus", " ")

```
*Figure 13.* Developmental period-specific enrichment levels and -coefficients of schizophrenia in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for schizophrenia in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure 14}
plot_agegroups("SCZ", "Prefrontal Cortex", " ")

```
*Figure 14.* Developmental period-specific enrichment levels and -coefficients of schizophrenia in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for schizophrenia in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

For schizophrenia, genes expressed in both the hippocampus (Figure 13) and prefrontal cortex (Figure 14) seems to rise over age in their enrichment, regardless of brain area. However this translated to merely one statistically significant enrichment coefficient. Only for genes expressed in the hippocampus at adolescence a statistically significant enrichment coefficient was found (coefficient = `r SCZ_ADO_HIP$Coefficient %>% formatC(format = "e", digits = 2)`, se = `r SCZ_ADO_HIP$Coefficient_std_error %>% formatC(format = "e", digits = 2)`, *P* = `r SCZ_ADO_HIP$Coefficient_p %>% formatC(format = "e", digits = 2)`). This was the only significant enrichment coefficient found for any disorder. Some of the developmental groups as indicated by the technical white paper of the BrainSpan Atlas of the Developing Human Brain have been collapsed into larger groups due to relatively little available data per group as defined by the techincal whitepaper. This may have caused some loss of signal, however the lack of statistically significant enrichment coefficients may also reflect the lack of available data even after collapsing developmental groups.
```{r  figure 15}
plot_agegroups("NEUdep", "Hippocampus", " ")

```
*Figure 15.* Developmental period-specific enrichment levels and -coefficients of neuroticism depressed affect in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure 16}
plot_agegroups("NEUdep", "Prefrontal Cortex", " ")

```
*Figure 16.* Developmental period-specific enrichment levels and -coefficients of neuroticism depressed affect in the prefrontal cortex. The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

Both genes expressed in the hippocampus (Figure 15) and the prefrontal cortex (Figure 16) at early prenatal, infancy, and adulthood periods showed statistically significant enrichment in their effect on neuroticism depressed affect. Differential enrichment between developmetal periods was found as well. Whereas the hippocampus showed enrichment at the middle prenatal period, the prefrontal cortex did not yield statistical significance for this developmental period. This is partly in line with findings of @Nagel2018, who did find an association of neuroticism with genes predominantly expressed in, amongst others, the frontal cortex. Their conditional analysis of tissue-specific expression showed general involvement of frontal cortex-expressed genes. However their study neglected to find any significant association with genes expressed in the hippocampus. These enrichments failed to result in any significant coefficients, none were found for neither hippocampus nor prefrontal cortex. 
In addition, it's notable that enrichment of disorders at a certain developmental period does not seem to be affected by the specific brain area, as only neuroticism depressed affect showed differing enrichment P values for the middle prenatal period between the hippocampus and the prefrontal cortex. This may indicate lack of differences in enrichment altogether.
```{r figure 17}
age_enrichment_HGHT_control1 <- control_for_height(input_age_enrichment, Nse = 1)

age_height <- input_age_enrichment %>% 
  filter(trait == "HGHT")
age_height2 <- left_join(age_enrichment_HGHT_control1, age_height, by = "Category")[,c(1, 21:29, 11:20)]

age_height2$Legend <- "Height - control"

age_enrichment_HGHT_control1$Legend <- age_enrichment_HGHT_control1$trait
age_enrichment_HGHT_control1$Legend <- str_replace_all(age_enrichment_HGHT_control1$Legend, 
                                              c("BIP" = "Bipolar disorder",
                                                "MDD" = "Major depression",
                                                "NEUdep" = "Neuroticism depressed affect", 
                                                "NEUwor" = "Neuroticism worry",
                                                "SCZ" = "Schizophrenia"))

colnames(age_height2) <- colnames(age_enrichment_HGHT_control1)
selection <- rbind(age_enrichment_HGHT_control1, age_height2)
remove(age_height, age_height2)
colours <- c(
    "Height - control" = "yellow",
    "Bipolar disorder" = "green",
    "Major depression" = "orange",
    "Neuroticism depressed affect" = "blue",
    "Neuroticism worry" = "purple",
    "Schizophrenia" = "red")

selection %>%
  unite("Disorder - Age group - Tissue", c(trait, cell_type_clean, trait_category), sep = " - ") %>%
  ggplot(mapping = aes(str_wrap(`Disorder - Age group - Tissue`, 30), Enrichment, group = 1)) +
    geom_point(alpha = 1, 
               size = 3,
               aes(color = Legend),
               show.legend = TRUE) +
    geom_errorbar(aes(ymin = Enrichment - Enrichment_std_error, 
                      ymax = Enrichment + Enrichment_std_error), 
                  width = .25,
                  alpha = .4) +
  scale_colour_manual(
      values = colours,
      limits = names(colours)) +
    ggtitle("") +
    theme(plot.title = element_text(size = 10, 
                                    face="bold", 
                                    margin = margin(10, 0, 10, 0))) +
    labs(x = "Disorder per developmental period for each brain area", 
         y = "Enrichment") +
    theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1, size = 7)) +
    theme(panel.background = element_rect(fill = 'Grey90')) +
  theme(legend.text = element_text(size = 8))

```
*Figure 17.* Developmental period-, and tissue-specific enrichment levels of disorders differing from developmental period-, and tissue-specific enrichment levels of height. Colours are indicative of specified disorders.

In addition to schizophrenia, some developmental periods significantly enriched in specific brain areas were also found for bipolar disorder (Figures S21 and S22), major depresson (Figures S27 and S28), neuroticism depressed affect (Figures 15 and 16), neuroticism worry (Figures S31 and S32), and lifetime cannabis use (Figures S23 and S24). However for these last five disorders no significant enrichment coefficients for a specific developmental period or brain area were found (see Supplementary Figures in Appendix II). The genes expressed in these developmental periods and brain areas are therefore ambiguous as to the presence of a true effect on these disorders. In order to estimate which of these developmental period-, and brain-specific enriched disorders might have an enrichment coefficient which failed to register, enrichment levels for height were used as a negative control (Figure 17). This resulted in a somewhat different picture for developmental period-, and brain-specific enrichment levels of disorders. When enrichment levels for height were used as a negative control, 14 developmental period- and tissue-specific groups showed differences with enrichment levels for height while not generating a significant enrichment coefficient. These developmental period-, and tissue-specific enrichment levels showed no overlap in their standard errors with those of enrichment levels for height. For genes expressed in the hippocampus at the early prenatal (enrichment = `r round(age_enrichment_HGHT_control1[1,5], 3)`, se = `r round(age_enrichment_HGHT_control1[1,6], 3)`, *P* = `r age_enrichment_HGHT_control1[1,7]`) and infancy (enrichment = `r round(age_enrichment_HGHT_control1[2,5], 3)`, se = `r round(age_enrichment_HGHT_control1[2,6], 3)`, *P* = `r age_enrichment_HGHT_control1[2,7]`) stages which were enriched in their effect on bipolar disorder, no overlap was shown with genes expressed in corresponding tissues and developmental periods which are enriched in their effect on height. For the prefrontal cortex, these developmental periods showed similar results (early prenatal: enrichment = `r round(age_enrichment_HGHT_control1[3,5], 3)`, se = `r round(age_enrichment_HGHT_control1[3,6], 3)`, *P* = `r age_enrichment_HGHT_control1[3,7]`; infancy: enrichment = `r round(age_enrichment_HGHT_control1[4,5], 3)`, se = `r round(age_enrichment_HGHT_control1[4,6], 3)`, *P* = `r age_enrichment_HGHT_control1[4,7]`). For genes expressed in both the hippocampus (enrichment = `r round(age_enrichment_HGHT_control1[13,5], 3)`, se = `r round(age_enrichment_HGHT_control1[13,6], 3)`, *P* = `r age_enrichment_HGHT_control1[13,7]`) and the prefrontal cortex (enrichment = `r round(age_enrichment_HGHT_control1[14,5], 3)`, se = `r round(age_enrichment_HGHT_control1[14,6], 3)`, *P* = `r age_enrichment_HGHT_control1[14,7]`) at adulthood, enriched in their effect on schizophrenia, showed no overlap with genes expressed in corresponding tissues and developmental periods which are enriched in their effect on height.

Whereas bipolar disorder and schizophrenia showed similar results for both hippocampus and prefrontal cortex, some enrichment levels unique to their brain area and developmental stage were found as well. Major depression showed distinct differences with height for enrichment levels in the prefrontal cortex at infancy (enrichment = `r round(age_enrichment_HGHT_control1[5,5], 3)`, se = `r round(age_enrichment_HGHT_control1[5,6], 3)`, *P* = `r age_enrichment_HGHT_control1[5,7]`). Furthermore, neuroticism depressed affect showed distinct differences with height for enrichment levels in the prefrontal cortex at the middle prenatal stage (enrichment = `r round(age_enrichment_HGHT_control1[8,5], 3)`, se = `r round(age_enrichment_HGHT_control1[8,6], 3)`, *P* = `r age_enrichment_HGHT_control1[8,7]`).

Another interesting observation is the difference in developmental stages for neuroticism depressed affect and neuroticism worry. Even though enrichment levels of these disorders corrected for height enrichment levels don't seem to differ over brain areas, these two subclusters of neuroticism differ in the developmental stages at which they were enriched. Genes expressed in both hippocampus (enrichment = `r round(age_enrichment_HGHT_control1[6,5], 3)`, se = `r round(age_enrichment_HGHT_control1[6,6], 3)`, *P* = `r age_enrichment_HGHT_control1[6,7]`) and prefrontal cortex (enrichment = `r round(age_enrichment_HGHT_control1[7,5], 3)`, se = `r round(age_enrichment_HGHT_control1[7,6], 3)`, *P* = `r age_enrichment_HGHT_control1[7,7]`) at the early prenatal stage were enriched in their effect on neuroticism depressed affect. However genes were uniquely expressed in the prefrontal cortex at the middle prenatal stage in their effect on the same disorder. Genes expressed in both hippocampus and prefrontal cortex at infancy (hippocampus: enrichment = `r round(age_enrichment_HGHT_control1[9,5], 3)`, se = `r round(age_enrichment_HGHT_control1[9,6], 3)`, *P* = `r age_enrichment_HGHT_control1[9,7]`; prefrontal cortex: enrichment = `r round(age_enrichment_HGHT_control1[11,5], 3)`, se = `r round(age_enrichment_HGHT_control1[11,6], 3)`, *P* = `r age_enrichment_HGHT_control1[11,7]`) and adulthood (hippocampus: enrichment = `r round(age_enrichment_HGHT_control1[10,5], 3)`, se = `r round(age_enrichment_HGHT_control1[10,6], 3)`, *P* = `r age_enrichment_HGHT_control1[10,7]`; prefrontal cortex: enrichment = `r round(age_enrichment_HGHT_control1[12,5], 3)`, se = `r round(age_enrichment_HGHT_control1[12,6], 3)`, *P* = `r age_enrichment_HGHT_control1[12,7]`) were enriched in their effect on neuroticism worry. This highlights further differences in genetic bases between the two neuroticism subclusters. However it should be noted that these results stem merely from examining overlap, or lack thereof, between the standard error-intervals around developmental period-, and tissue-specific enrichment levels of disorders and their corresponding enrichment levels of height. When standard errors do not overlap, there is no assurance that the difference between these enrichment levels is statistically significant. Nonetheless, the genes expressed in the developmental periods and brain areas which qualify for this condition, might be interesting candidates for further in-depth study relating to the presence of a true effect on relevant disorders.

\newpage

# Conclusion

This study aimed to identify genetic influences on psychiatric and neurological disorders, indicating pleiotropy of contributing loci. To this end, first genetic correlations were computed between all psychiatric and neurological disorders. Genetic correlations found were largely concordant with those of @Lee2019, with some new insights that were found due to the addition of other psychiatric and neurological disorders. Most notable of these were the strong genetic relationships between post-traumatic stress disorder, major depression and neuroticism depressed affect. We further found a relationship between post-traumatic stress disorder, major depression and neuroticism worry. However the former was found to be stronger than the latter. A strong genetic relationship between neuroticism depressed affect and neuroticism worry was found as well. In addition, some of the differential relationships for neuroticism depressed affect and neuroticism worry with other disorders have been replicated from earlier studies. @Nagel2018 had found relationships for neuroticism depressed affect with attention-deficit/hyperactivity disorder and for neuroticism worry with anorexia nervosa, bipolar disorder, and schizophrenia. These were found as well in the current study. A newly found result howwever is the positive relationship between neuroticism depressed affect and anorexia nervosa. This relationship is weaker than the relationship between neuroticism worry and anorexia nervosa. Nonetheless this suggests that neuroticism worry is not unique as a neuroticism subcluster in its relationship with anorexia nervosa. The differences between neuroticism depressed affect and neuroticism worry in their genetic correlations with other disoders and traits may be indicative of differences in genetic foundation between the neuroticism subclusters and the overarching personality trait neuroticism. Neuroticism subclusters may be examined and, possibly, be redefined based upon their genetic communalities and differences rather than on their phenotypical communalities and differences. This raises the question how valid established subclusters of neuroticism specifically and the big five personality traits in general really are when evaluating them using genetic analysis.

Overall, results were largely concordant with those of @Lee2019. However some discordant results were found as well. No significant relationship was found between anorexia nervosa and attention-deficit/hyperactivity disorder, whereas @Lee2019 had found a negative genetic association. These differences are most likely attributable to the use of GWAS studies with higher numbers of participants.

Elaborating on some of the newly generated insights and the degree to which they were concordant with expectations. A moderate genetic relationship was found for lifetime cannabis use and schizophrenia. This lends further validity to previous studies showing positive correlations between genetic risk factors for cannabis use and schizophrenia [Verweij, et al., 2017, Power, et al., 2014; cited by @Pasman2018], and suggests a positive relationship between genetic risk factors for these phenotypes. Furthermore, relationships were found for post-traumatic disorder with schizophrenia and major depression. Earlier research suggested a strong overlapping genetic risk for post-traumatic stress disorder with schizophrenia and a moderate overlapping genetic risk with major depression [@Duncan2018]. However these results suggest a stronger genetic relationship of post-traumatic stress disorder with major depression than with schizophrenia. A possible explanation might arise examining the low amount of risk loci identified (2, see Table 1) for post-traumatic stress disorder as of yet [@Nievergelt2019]. LDSC software explicitly models LD rather than interpreting individual risk genes [@Finucane2015]. Nonetheles, the low amount of risk loci identified may suggest a lack of statistical power to detect genetic risk. The lack of power to detect genetic risk might skew results, which in turn may explain the discrepancy in genetic correlations found between this study and that of @Duncan2018. Regarding lifetime cannabis use, other than the afore mentioned genetic correlation with schizophrenia, relationships were found with attention-deficit/hyperactivity disorder, anorexia nervosa,  autism spectrum, bipolar disorder, major, and neuroticism. These genetic correlations highlight the possible role lifetime cannabis use may play in the development of these disorders and merits further study of the underlying relationships.

Other relationships which were expected, failed to be replicated in the genetic correlations. Autism spectrum disorder did not show any significant genetic correlations with bipolar disorder or anorexia nervosa. Nonetheless relationships which fulfilled expectations for autism spectrum disorder were found for its relationships with schizophrenia, major depression and attention-deficit/hyperactivity disorder. Insomnia did not show the expected genetic relationship with lifetime cannabis use. However genetic correlations of insomnia with attention-deficit/hyperactivity disorder, major depression, neuroticism depressed affect, neuroticism worry, and post-traumatic stress disorder were found. For Alzheimer's disease, no pleiotropy was found at all. The lack of genetic relationships found for Alzheimer's disease might be due to heterogeneity within the disorder itself. @Mukherjee2019 identified subcategories of Alzheimer's disease. These subcategories were differentiated into cognitively-defined subgroups on the basis of relative performance in memory, executive functioning, visuospatial functioning and language, and may each have a unique genetic basis. The study lacked sufficient power to identify SNPs that convey risk for these subgroups. However the data support the biological coherence of cognitively defined subgroups and identified candidate genetic loci with extreme odds ratios (ORs). Hence there might be a relationship between specific brain regions and subgroups of Alzheimer’s disease. These subgroups’ differential performances in these fields might indicate some specific expression profiles in brain regions related to these fields, which in turn might overlap with psychiatric disorders showing similar symptoms. A GWAS with a larger population sample might have sufficient power to clearly identify these subgroups. These subgroups in turn could show uniquely identifiable signals of pleiotropy which may be masked at present.

Secondly, this study aimed to discover shared genetic bases between psychiatric and neurological disorders by connecting human genomic findings to life phase-, cell type- and tissue-specific RNA transcriptome expression. Regarding cell type-specific enrichment, genes were expressed with a significant enrichment coefficient in specific cell types for anorexia nervosa (Unclassified), major depression (MG), neuroticism depressed affect (GABA2), neuroticism worry (exPFC1, GABA2), and schizophrenia (exPFC1, GABA2). Especially the negative enrichment coefficient of MG on major depression is noteworthy, since it may shine a light on the as of yet ambiguous role of microglia on major depression in earlier research [@Perry2018]. For height, all cel types showed significant enrichment except for exPFC1, however no significant enrichment coefficient was found for any of those cell types. Some specific cell types were enriched in their effect on disorders, however they were lacking a significant enrichment coefficient. These findings may still indicate similarities and differences in genetic foundation between disorders. 

In this regard, genes expressed in END, MG, and ODC1 showed depletion in their effect on neuroticism depressed affect. This depletion suggests there is less heritability for END, MG, and ODC1 on neuroticism depressed affect than can be expected based on the size of the annotation. Genes expressed in exCA3 and exPFC1 in turn, showed enrichment in their effect on neuroticism worry. The differences in genes expressed in cell types enriched in their effect on the neuroticism subclusters, generate further evidence for the existence of genetic differences between these neuroticism subclusters. However some common genetic foundation was found as well. Genes expressed in exCA1 and GABA2 showed rougly similar enrichment levels in their effect on both neuroticism subclusters. Regarding the enrichment coefficients, this observation holds up for GABA2. When comparing both neuroticism subclusters to major depression, similarities arise as well. All three phenotypes show high levels of enrichment in GABA2. Furthermore, genes expressed in MG showed depletion in their effect on both neuroticism depressed affect and major depression. In contrast, MG is not significantly enriched for neuroticism worry. The depleted enrichment levels suggest there is less heritability for MG on both major depression and neuroticism deppressed affect than can be expected based on the size of the annotation. This may indicate that enrichment of MG plays a role in the differences in the genetic correlations found between major depression and neuroticism depressed affect as opposed to that between major depression and neuroticism worry.

For schizophrenia a multitude of cell types showed enrichment, however only exPFC1 turned out to have a significant enrichment coefficient. Similarities in genetic basis between schizophrenia and neuroticism worry might in part be explained by positive enrichment coefficients of exPFC1 and GABA2 on both disorders. These cell type expression profiles may lie at the root of the common genetic basis between the two disorders. In addition, some enriched cell types were also found for Alzheimer's disease (ASC2, END, exCA1, exDG, exPFC1, MG, NSC, ODC1, and Unclassified), bipolar disorder (exCA1, exCA3, exDG, exPFC1, GABA1, GABA2, OPC, and Unclassified), and insomnia (GABA2). However these last three disorders did not yield any significant enrichment coefficients for a specific cell type. These cell types are therefore ambiguous as to the presence of true genetic effects on these disorders.

In addition to anorexia nervosa, major depression, neuroticism depressed affect, neuroticism worry, and schizophrenia, some significantly enriched cell types were also found for Alzheimer's disease, bipolar disorder, and insomnia. However for these last three disorders no significant enrichment coefficients for a specific cell type were found. The genes expressed in these cell types are therefore ambiguous as to the presence of a true effect on their respective disorders. In order to estimate which of these cell type-specific enriched disorders might have an enrichment coefficient which failed to register, enrichment levels for height were used as a negative control. This resulted in a somewhat different picture for cell type-specific enrichment levels of disorders. Of the cell types enriched in their effect on respective disorders that failed to generate a significant enrichment coefficient, six cell types did not show an overlap of the enrichment level standard error with that of height. Four for Alzheimer's disease (END, exCA1, exDG, and Unclassified), one for neuroticism depressed affect (MG), and one for schizophrenia (Unclassified). These results may indicate that these cell type-specific enrichments have an effect on their respective disorders, but nonetheless that sample sizes are too small to detect a enrichment coefficient. However it should be noted that these results stem merely from examining overlap, or lack thereof, between the standard error-intervals around cell type-specific enrichment levels of disorders and their corresponding enrichment levels of height. When standard errors do not overlap, there is no assurance that the difference between these enrichment levels is statistically significant. Nonetheless, the genes expressed in the cell types which qualify for this condition, might be interesting candidates for further in-depth study relating to the presence of a true effect on relevant disorders.

Regarding life phase- and tissue-specific enrichment. When studying the enrichment levels, a clear network of relationships with significant overlap in originating developmental periods between several disorders. However when looking for coefficients of these enrichments, only a relationship between adolescence and hippocampal schizophrenia remained. Height showed significant enrichment for all developmental period, however no significant enrichment coefficient was found. For schizophrenia, enrichment levels in both the hippocampus and prefrontal cortex seems to rise over age, regardless of brain area. However this only translated into one significant enrichment coefficient. Only for hippocampal expression of schizophrenia at adolescence a statistically significant effect was found. This was the only significant enrichment coefficient found for any disorder. For neuroticism depressed affect, both the prefrontal cortex and the hippocampus showed statistically significant enrichment for early prenatal, infancy, and adulthood periods for neuroticism depressed affect. Differential enrichment between developmental periods was found as well. Whereas the hippocampus showed enrichment at the middle prenatal period, the prefrontal cortex did not yield statistical significance for this developmental period. This is partly in line with findings of @Nagel2018, who found an association of neuroticism with genes predominantly expressed in, amongst others, the frontal cortex. However their study neglected to find any significant association with genes expressed in the hippocampus. 

When using enrichment levels for height as a negative control, of the developmental period- and tissue-specific groups per disorder that failed to generate a significant enrichment coefficient, 14 did not have an overlap of the enrichment level standard error with its corresponding enrichment level standard error interval for height. 

Most of the enrichment levels controlled for by the use of enrichment levels of height, showed similar results for the hippocampus and the prefrontal cortex. For bipolar disorder, the early prenatal and infancy stages showed no overlap with enrichment levels for height, regardless of brain area. For schizophrenia at adulthood no overlap was found either, again regardless of brain area. Taking into account that schizophrenia at the developmental phase preceding adulthood showed a significant enrichment coefficient, the argument could be made for an alternative subdivision of developmental stages. For example, early adulthood might be lumped in with the adolescence developmental phase. This might result in a enrichment coefficient of schizophrenia on this developmental phase for both hippocampus and prefrontal cortex. Some enrichment levels unique to their brain area and developmental stage were found as well. Major depression showed distinct differences with height for enrichment levels in the prefrontal cortex at infancy. Furthermore, neuroticism depressed affect showed distinct differences with height in the prefrontal cortex at the middle prenatal stage. 

Another interesting observation is the difference in developmental stages for neuroticism depressed affect and neuroticism worry. Even though enrichment levels of these disorders corrected for height enrichment levels don't seem to differ over brain areas, these two subclusters of neuroticism differ in the developmental stages at which they were found to be enriched. Genes expressed in both hippocampus and prefrontal cortex at the early prenatal stage were enriched in their effect on neuroticism depressed affect. However genes were uniquely expressed in the prefrontal cortex at the middle prenatal stage in their effect on the same disorder. Genes expressed in both hippocampus and prefrontal cortex at infancy and adulthood were enriched in their effect on neuroticism worry. This highlights further differences in genetic bases between the two neuroticism subclusters. There could be value in this for diagnostic research if a focus is put on the timing of development for disorders which correlate differentially with the two neuroticism subclusters.

## Limitations

One limitation of this study is that LDSC estimates should be interpreted with some caution. Genetic correlations estimated by LDSC are known to be less accurate compared to the genomic restricted maximum likelihood (GREML) approach, even when smaller population samples are available for GREML than for LDSC. @Ni2018 showed that there can be uncertainty about homogeneity among combined meta-datasets and should be followed up with analyses using GREML methods. 

Another limitation is the lack of sufficient data for all developmental periods as defined by the technical white paper of the BrainSpan Atlas of the Developing Human Brain. In order to have sufficient data for all developmental periods, some adjacent developmental periods were collapsed into one. This caused the number of developmental periods to shrink from 13 to 7 and could have resulted in loss of signal. The lack of statistically significant enrichment coefficients may also reflect the lack of available data even after collapsing several developmental periods.

Furthermore, differential expression analysis was used to distinguish between the different developmental periods for RNA-Seq (v10) summarized to genes. This software was designed to process un-normalized counts, however normalized counts were used for the current study. The subsequent inability to process decimals in the normalized counts was corrected for by rounding these counts to full numbers. This may have caused some of the variation to diminish, even though the descriptives seem to suggest no differences exist between the normalized counts pre- and after correction. An alternative solution might have been to multiply all counts sufficiently that no decimals remained in the normalized counts. Neither of these solutions would have been optimal though, since the DESeq2 model corrects internally for library size (Love, Huber, & Anders, 2014). Ergo, transformed or normalized values are not ideal as input.

In addition, some of the population prevalences in Table 1 should be considered only rough estimates with limited validity. Some of these population prevalences were possibly influenced by available measurement tools and exact definitions (post-traumatic stress disorder), national governments policies (lifetime cannabis use), or highly dependent on the age distribution of a population (Alzheimer’s disease). This may have caused the calculation of the liability heritability estimates to be affected. These limitations were further elaborated on in the method section.

A final limitation to this study is due to the use of height as a negative control measure for enrichment levels of cell type-, brain-, and developmental period-specific enrichment of disorders. Height was found to be significantly enriched for almost all cell types (all but exPFC1) and all developmental periods regardless of brain area. This calls into question the validity of enrichment levels without a significant coefficient as an estimate for common or differential genetic basis. Enrichment levels alone may not be a valid ground to draw conclusions regarding genetic bases for disorders and their common foundation with other disorders. It may however, offer up candidates for further study regarding their exact role in the expression of disorders and similarities or dissimilarities in expression between disorders.

In closing, a large number of genetic influences on psychiatric and neurological disorders were identified through genetic correlations and the life phase-, cell type- and tissue-specific RNA transcriptome generated further insight in their expression. Found results may hold implications for diagnostic definitions of disorders and put into question how valid these and other phenotypes are if solely based upon observational research.

\newpage

# References
```{r create_r-references}
r_refs(file = "r-references.bib")


```

\begingroup
\setlength{\parindent}{-0.5in}
\setlength{\leftskip}{0.5in}

<div id="refs" custom-style="Bibliography"></div>
\endgroup

\newpage

# Appendix I

Not all cell type-specific enrichments of disorders yielded significant enrichment levels and/or -coefficients. Therefore not all were included in the text. In this appendix, cell type-specific enrichments of all disorders have been included.

```{r figure S1}
plot_celltype("ADHD", " ")

```
*Figure S1.* Cell type-specific enrichment levels and -coefficients of attention-deficit/hyperactivity disorder. The figures on the left portray enrichment levels, and -coefficients for attention-deficit/hyperactivity disorder. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S2}
plot_celltype("ALZ", " ")

```
*Figure S2.* Cell type-specific enrichment levels and -coefficients of Alzheimer’s disease. The figures on the left portray enrichment levels, and -coefficients for Alzheimer’s disease. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S3}
plot_celltype("AN", " ")

```
*Figure S3.* Cell type-specific enrichment levels and -coefficients of anorexia nervosa. The figures on the left portray enrichment levels, and -coefficients for anorexia nervosa. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S4}
plot_celltype("ASD", " ")

```
*Figure S4.* Cell type-specific enrichment levels and -coefficients of autism spectrum disorder. The figures on the left portray enrichment levels, and -coefficients for autism spectrum disorder. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S5}
plot_celltype("BIP", " ")

```
*Figure S5.* Cell type-specific enrichment levels and -coefficients of bipolar disorder. The figures on the left portray enrichment levels, and -coefficients for bipolar disorder. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S6}
plot_celltype("CAN", " ")

```
*Figure S6.* Cell type-specific enrichment levels and -coefficients of lifetime cannabis use. The figures on the left portray enrichment levels, and -coefficients for lifetime cannabis use. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S7}
plot_celltype("INS", " ")

```
*Figure S7.* Cell type-specific enrichment levels and -coefficients of insomnia. The figures on the left portray enrichment levels, and -coefficients for insomnia. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S8}
plot_celltype("MDD", " ")

```
*Figure S8.* Cell type-specific enrichment levels and -coefficients of major depression. The figures on the left portray enrichment levels, and -coefficients for major depression. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S9}
plot_celltype("NEUdep", " ")

```
*Figure S9.* Cell type-specific enrichment levels and -coefficients of neuroticism depressed affect. The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S10}
plot_celltype("NEUwor", " ")

```
*Figure S10.* Cell type-specific enrichment levels and -coefficients of neuroticism worry. The figures on the left portray enrichment levels, and -coefficients for neuroticism worry. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S11}
plot_celltype("PTSD", " ")

```
*Figure S11.* Cell type-specific enrichment levels and -coefficients of post-traumatic stress disorder. The figures on the left portray enrichment levels, and -coefficients for post-traumatic stress disorder. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r figure S12}
plot_celltype("SCZ", " ")

```
*Figure S12.* Cell type-specific enrichment levels and -coefficients of schizophrenia. The figures on the left portray enrichment levels, and -coefficients for schizophrenia. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

\newpage

# Appendix II

Not all developmental period- and tissue-specific enrichments of disorders yielded significant enrichment levels and/or -coefficients. Therefore not all were included in the text. In this appendix, developmental period- and tissue-specific enrichments of all disorders have been included.

```{r  figure S13}
plot_agegroups("ADHD", "Hippocampus", " ")

```
*Figure S13.* Developmental period-specific enrichment levels and -coefficients of attention-deficit/hyperactivity disorder in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for attention-deficit/hyperactivity disorder in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S14}
plot_agegroups("ADHD", "Prefrontal Cortex", " ")

```
*Figure S14.* Developmental period-specific enrichment levels and -coefficients of attention-deficit/hyperactivity disorder in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for attention-deficit/hyperactivity disorder in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S15}
plot_agegroups("ALZ", "Hippocampus", " ")

```
*Figure S15.* Developmental period-specific enrichment levels and -coefficients of Alzheimer’s disease in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for Alzheimer’s disease in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S16}
plot_agegroups("ALZ", "Prefrontal Cortex", " ")

```
*Figure S16.* Developmental period-specific enrichment levels and -coefficients of Alzheimer’s disease in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for Alzheimer’s disease in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S17}
plot_agegroups("AN", "Hippocampus", " ")

```
*Figure S17.* Developmental period-specific enrichment levels and -coefficients of anorexia nervosa in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for anorexia nervosa in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S18}
plot_agegroups("AN", "Prefrontal Cortex", " ")

```
*Figure S18.* Developmental period-specific enrichment levels and -coefficients of anorexia nervosa in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for anorexia nervosa in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S19}
plot_agegroups("ASD", "Hippocampus", " ")

```
*Figure S19.* Developmental period-specific enrichment levels and -coefficients of autism spectrum disorder in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for autism spectrum disorder in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S20}
plot_agegroups("ASD", "Prefrontal Cortex", " ")

```
*Figure S20.* Developmental period-specific enrichment levels and -coefficients of autism spectrum disorder in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for autism spectrum disorder in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S21}
plot_agegroups("BIP", "Hippocampus", " ")

```
*Figure S21.* Developmental period-specific enrichment levels and -coefficients of bipolar disorder in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for bipolar disorder in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S22}
plot_agegroups("BIP", "Prefrontal Cortex", " ")

```
*Figure S22.* Developmental period-specific enrichment levels and -coefficients of bipolar disorder in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for bipolar disorder in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S23}
plot_agegroups("CAN", "Hippocampus", " ")

```
*Figure S23.* Developmental period-specific enrichment levels and -coefficients of lifetime cannabis use in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for lifetime cannabis use in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S24}
plot_agegroups("CAN", "Prefrontal Cortex", " ")

```
*Figure S24.* Developmental period-specific enrichment levels and -coefficients of lifetime cannabis use in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for lifetime cannabis use in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S25}
plot_agegroups("INS", "Hippocampus", " ")

```
*Figure S25.* Developmental period-specific enrichment levels and -coefficients of insomnia in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for insomnia in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S26}
plot_agegroups("INS", "Prefrontal Cortex", " ")

```
*Figure S26.* Developmental period-specific enrichment levels and -coefficients of insomnia in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for insomnia in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S27}
plot_agegroups("MDD", "Hippocampus", " ")

```
*Figure S27.* Developmental period-specific enrichment levels and -coefficients of major depression in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for major depression in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S28}
plot_agegroups("MDD", "Prefrontal Cortex", " ")

```
*Figure S28.* Developmental period-specific enrichment levels and -coefficients of major depression in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for major depression in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S29}
plot_agegroups("NEUdep", "Hippocampus", " ")

```
*Figure S29.* Developmental period-specific enrichment levels and -coefficients of neuroticism depressed affect in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S30}
plot_agegroups("NEUdep", "Prefrontal Cortex", " ")

```
*Figure S30.* Developmental period-specific enrichment levels and -coefficients of neuroticism depressed affect in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for neuroticism depressed affect in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S31}
plot_agegroups("NEUwor", "Hippocampus", " ")

```
*Figure S31.* Developmental period-specific enrichment levels and -coefficients of neuroticism worry in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for neuroticism worry in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S32}
plot_agegroups("NEUwor", "Prefrontal Cortex", " ")

```
*Figure S32.* Developmental period-specific enrichment levels and -coefficients of neuroticism worry in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for neuroticism worry in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S33}
plot_agegroups("PTSD", "Hippocampus", " ")

```
*Figure S33.* Developmental period-specific enrichment levels and -coefficients of post-traumatic stress disorder in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for post-traumatic stress disorder in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S34}
plot_agegroups("PTSD", "Prefrontal Cortex", " ")

```
*Figure S34.* Developmental period-specific enrichment levels and -coefficients of post-traumatic stress disorder in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for post-traumatic stress disorder in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.


```{r  figure S35}
plot_agegroups("SCZ", "Hippocampus", " ")

```
*Figure S35.* Developmental period-specific enrichment levels and -coefficients of schizophrenia in the hippocampus. The figures on the left portray enrichment levels, and -coefficients for schizophrenia in the hippocampus. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.

```{r  figure S36}
plot_agegroups("SCZ", "Prefrontal Cortex", " ")

```
*Figure S36.* Developmental period-specific enrichment levels and -coefficients of schizophrenia in the prefrontal cortex The figures on the left portray enrichment levels, and -coefficients for schizophrenia in the prefrontal cortex. The figure on the right portrays corresponding enrichment levels of height, which is used as a negative control. Blue dots indicate significant enrichment levels, and -coefficients at FDR < 0.05, red dots indicate non-significant enrichment levels, and -coefficients.


